webapp/lib/platform.dart

```diff
@@ -59,6 +59,19 @@ bool isUserSignedIn() {
 
 typedef CollectionListener(List<DocSnapshot> changes);
 
+Future<List<String>> get activeProjects async =>
+    (await _firestoreInstance.collection('projects').where('archived', '==', false).get()).docs.map((doc) => doc.id).toList();
+
+Future<Map<String, List<String>>> get getProjectsDrivers async {
+  Map<String, List<String>> projectsDrivers = {};
+  for (var project in (await activeProjects)) {
```

Comment:
we end up doing 2 `await activeProjects` calls. I'm assuming the result is almost instant? And chances of getting different results are quite minimal, and easily fixed via a refresh, so let's leave it as it is for now, but we might consider in the future cacheing the result instead of making another call to firebase.

Reply:
Yes. The calls are instant. Noted